src/jv_aux.c

```diff
@@ -151,13 +151,16 @@ jv jv_get(jv t, jv k) {
     jv_free(t);
     jv_free(k);
   }
+
+  if(jv_is_borrowed(v)) return jv_unborrow(v);
```

Comment:
This is superfluous.  The value we're returning is necessarily already not borrowed.